ts/input/tex/ParseUtil.ts

```diff
@@ -64,63 +230,52 @@ namespace ParseUtil {
    *     unit name, length of matched string. The latter is interesting in the
    *     case of trailing garbage.
    */
-  export function matchDimen(
+  matchDimen: function(
     dim: string, rest: boolean = false): [string, string, number] {
-      let match = dim.match(rest ? dimenRest : dimenEnd);
-      return match ?
-        muReplace([match[1].replace(/,/, '.'), match[4], match[0].length]) :
-        [null, null, 0];
-  }
-
-
-  /**
-   * Transforms mu dimension to em if necessary.
-   * @param {[string, string, number]} [value, unit, length] The dimension triple.
-   * @return {[string, string, number]} [value, unit, length] The transformed triple.
-   */
-  function muReplace([value, unit, length]: [string, string, number]): [string, string, number] {
-    if (unit !== 'mu') {
-      return [value, unit, length];
-    }
-    let em = Em(UNIT_CASES[unit](parseFloat(value || '1')));
-    return [em.slice(0, -2), 'em', length];
-  }
+    let match = dim.match(rest ? ParseUtil.UNIT_CASES.dimenRest : ParseUtil.UNIT_CASES.dimenEnd);
+    return match ?
+      muReplace([match[1].replace(/,/, '.'), match[4], match[0].length]) :
+      [null, null, 0];
+  },
 
 
   /**
    * Convert a dimension string into standard em dimension.
    * @param {string} dim The attribute string.
    * @return {number} The numerical value.
    */
-  export function dimen2em(dim: string): number {
-    let [value, unit] = matchDimen(dim);
+  dimen2em: function(dim: string): number {
```

Comment:
This could be done as 

``` js
  dimen2em(dim: string): number) {
```

for a savings of 10 characters per function.  Not huge, but worth considering, as it does add up, when you consider doing the same with the other functions below.

Reply:
Done.